src/main/java/com/teragrep/cfe18/handlers/FileProcessingTypeController.java

```diff
@@ -150,15 +149,12 @@ public ResponseEntity<String> newFileProcessingType(@RequestBody FileProcessing
                     newFileProcessing.getInputvalue());
             LOGGER.debug("Values returned <[{}]>",n);
             JSONObject jsonObject = new JSONObject();
-            // ID is never returned from database so null should suffice.
-            String v = null;
-            jsonObject.put("id", v);
             jsonObject.put("message", "New file processing type created with the name = " + n.getName());
 
             return new ResponseEntity<>(jsonObject.toString(), HttpStatus.CREATED);
         } catch (RuntimeException ex) {
             JSONObject jsonErr = new JSONObject();
-            jsonErr.put("id", 0);
+            jsonErr.put("id", newFileProcessing.getId());
```

Comment:
there is none as id is allocated only in insert, name suggest that this is an insert operation. please remove